### PR TITLE
Last Test Objectives -- SESS, INPV, BUSL

### LINKED ISSUE #602: Set Testing Objectives For All Test Scenarios
**What's the issue?**
Not all testing scenarios have a clear and concise objective(s)

**How do we solve it?**
Go over the WSTG and ensure that every test scenario has clear objectives in a UL format (using dashes).

Since this is one of the bigger issues and will cover lots of scenarios, I'll tackle it and make sure to make it fast.

### PR DESCRIPTION
**Pull request by ThunderSon**
This PR closes #602 

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Set all SESS test objectives that were missing.
- Set BUSL test objectives.
- Format INPV tests objectives

### CODE REVIEW

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/03-Test_Integrity_Checks.md [hunk 1]
```diff
@@ -32,13 +30,14 @@ Suppose the motor vehicle administration system required an employee initially v
 
 Many systems include logging for auditing and troubleshooting purposes. But, how good/valid is the information in these logs? Can they be manipulated by attackers either intentionally or accidentally having their integrity destroyed?
 
-## How to Test
+## Test Objectives
 
-### Generic Testing Method
+- Review the project documentation for components of the system that move, store, or handle data.
+- Determine what type of data is logically acceptable by the component and what types the system should guard against.
+- Determine who should be allowed to modify or read that data in each componenet
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/03-Test_Integrity_Checks.md:37]:
```suggestion
- Determine who should be allowed to modify or read that data in each component.
```

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/03-Test_Integrity_Checks.md [hunk 2]
```diff
@@ -32,13 +30,14 @@ Suppose the motor vehicle administration system required an employee initially v
...
+- Attempt to insert, update, or edit delete the data values with invalid data into each component by users that should not be allowed per the business logic workflow.
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/03-Test_Integrity_Checks.md:38]:
```suggestion
- Attempt to insert, update, or delete data values used by each component that should not be allowed per the business logic workflow.
```

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/04-Test_for_Process_Timing.md [hunk 3]
```diff
@@ -30,11 +28,18 @@ Most Arenas or travel agencies have ticketing applications that allow users to p
 
 Suppose a precious metals e-commerce site allows users to make purchases with a price quote based on market price at the time they log on. What if an attacker logs on and places an order but does not complete the transaction until later in the day only of the price of the metals goes up? Will the attacker get the initial lower price?
 
+## Test Objectives
+
+- Review the project documentation for system functionality that may be impacted by time.
+- Develop and execute the mis-use cases.
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/04-Test_for_Process_Timing.md:34]:
```suggestion
- Develop and execute misuse cases.
```

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/05-Test_Number_of_Times_a_Function_Can_Be_Used_Limits.md [hunk 4]
```diff
@@ -12,10 +12,15 @@ Vulnerabilities related to testing for the function limits are application speci
...
 
-## Example
+### Example
 
 Suppose an eCommerce site allows users to take advantage of any one of many discounts on their total purchase and then proceed to checkout and tendering. What happens of the attacker navigates back to the discounts page after taking and applying the one “allowable” discount? Can they take advantage of another discount? Can they take advantage of the same discount multiple times?
 
+## Test Objectives
+
+- Identify functions that must set limits to the times they can be called.
+- Assess if their is a logical limit set on the functions and if it is properly validated.
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/05-Test_Number_of_Times_a_Function_Can_Be_Used_Limits.md:22]:
```suggestion
- Assess if there is a logical limit set on the functions and if it is properly validated.
```

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/06-Testing_for_the_Circumvention_of_Work_Flows.md [hunk 5]
```diff
@@ -25,19 +23,19 @@ Many of us receive so type of “club/loyalty points” for purchases from groce
 
 An electronic bulletin board system may be designed to ensure that initial posts do not contain profanity based on a list that the post is compared against. If a word on a deny list is found in the user entered text the submission is not posted. But, once a submission is posted the submitter can access, edit, and change the submission contents to include words included on the profanity/deny list since on edit the posting is never compared again. Keeping this in mind, attackers may open an initial blank or minimal discussion then add in whatever they like as an update.
 
-## How to Test
+## Test Objectives
 
-### Generic Testing Method
+- Review the project documentation for methods to skip or go to steps in the application process in a different order from the intended business logic flow.
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/06-Testing_for_the_Circumvention_of_Work_Flows.md:28]:
```suggestion
- Review the project documentation for methods to skip or go through steps in the application process in a different order from the intended business logic flow.
```

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/05-Test_Number_of_Times_a_Function_Can_Be_Used_Limits.md (hunk 4 above)
Comment (ThunderSon) [reply document/4-Web_Application_Security_Testing/10-Business_Logic_Testing/05-Test_Number_of_Times_a_Function_Can_Be_Used_Limits.md:22]: Oof that was bad. Surprised you still respect me.

**Comment by kingthorin** [comment]
> rick's lint

Hahahahahaha 😃

**Review by kingthorin — APPROVED** [review]

**Pull request merged**